extensions/amp-analytics/0.1/config.js

```diff
@@ -335,6 +335,27 @@ export function expandConfigRequest(config) {
   return config;
 }
 
+/**
+ * Reads default configuration object, excluding available overrides. Returns
+ * list of names of available overrides.
+ *
+ * @param {!JsonObject} config
+ * @param {!Object<string, *>} defaultConfig Stores default configuration with
+ * all overrides removed.
+ * @return {!Array<string>} availableOverrides
+ */
+export function readDefaultConfiguration(config, defaultConfig) {
```

Comment:
I don't quite like methods have output in both return and arguments.

how about this:

```
function mergeDefaultConfig(config) {
  // return a map of merged configs
}
```